### PR TITLE
Add a configurable about page

### LINKED ISSUE #323: Create a configurable "about" page
**Feature Category**

- [ ] Correctness
- [x] User Interface / User Experience
- [ ] Performance
- [ ] Other (please explain)

**Describe the problem**

Back in the day mquery used to have an about page (it's still visible in the gifs on the main page, even). It was removed, because it was not very useful. But now I (me, @msm-code) would like to put some customised information in my mquery instance. Therefore I'd like to have an about page that I can configure.

**Describe the solution you'd like**

I'd like to have an about page where I can put some of my links or explaination. I'm very flexible when it comex to the details:

* about page may be optional, or mandatory for every instance (with a sensible default)
* it may be configured via a config file, environment, special file on the server, or anything.
* It may be written in markdown, html, etc.

Not technically this issue, but an option to change the logo (from certpl to a customised one) would be nice too.

**Describe alternatives you've considered**

-

### PR DESCRIPTION
**Pull request by msm-code**
**Your checklist for this pull request**
- [x] I've read the [contributing guideline](https://github.com/CERT-Polska/mquery/blob/master/CONTRIBUTING.md).
- [x] I've tested my changes by building and running mquery, and testing changed functionality (if applicable)
- [ ] I've added automated tests for my change (if applicable, optional)
- [ ] I've updated documentation to reflect my change (if applicable)

**What is the current behaviour?**

There is no way to add any customised information for the users #323 

**What is the new behaviour?**
This PR adds (will add) a customisable "about" page. It's not very customisable, because all the information will have to fit into a config file, but that should be enough to give users a few URLs or explain what's going on.

image

This subpage is optional (will be optional in the final PR).

I've decided to allow HTML (it's trusted - it comes from the server) instead of just plain text.

**Test plan**

Visit /about.

**Closing issues**

fixes #323

### CODE REVIEW

**Comment by msm-code** [comment]
Rebased onto typed config. Ready to merge.

File: src/mqueryfront/src/Navigation.js [hunk 1]
```diff
@@ -33,59 +34,68 @@ function Navigation(props) {
...
-                type="button"
-                data-toggle="collapse"
-                data-target="#navbarSupportedContent"
-                aria-controls="navbarSupportedContent"
-                aria-expanded="false"
-                aria-label="Toggle navigation"
-            >
-                <span className="navbar-toggler-icon" />
-            </button>
+            <div className="container-fluid">
```
Comment (msm-code) [inline src/mqueryfront/src/Navigation.js:37]: This is a bit unrelated to the current PR - new bootstrap requires navbar contents to be contained within a conteiner-fluid div

**Review by nazywam — APPROVED** [review]

File: src/schema.py [hunk 2]
```diff
@@ -130,3 +130,4 @@ class ServerSchema(BaseModel):
     auth_enabled: Optional[str]
     openid_url: Optional[str]
     openid_client_id: Optional[str]
+    about: str
```
[the remark this answers is not in the document]
Comment (msm-code) [reply src/schema.py:133]: Yes. Originally this was `Optional[str]` in the schema too, but I realised this will never actually be null.
